spec/PeriodicBackgroundSync-index.bs

```diff
@@ -0,0 +1,95 @@
+<pre class='metadata'>
+Title: Web Periodic Background Synchronization
+Status: CG-DRAFT
+ED: https://wicg.github.io/BackgroundSync/spec/PeriodicBackgroundSync-index.bs
+Shortname: periodic-background-sync
+Level: 1
+Editor: Mugdha Lakhani, Google, nator@chromium.org
+Abstract: This specification describes a method that enables web applications to periodically synchronize data and content in the background.
+Group: wicg
+Repository: WICG/BackgroundSync
+Link Defaults: html (dfn) allowed to show a popup/event handler idl attribute/global object/in parallel/incumbent settings object/perform a microtask checkpoint/queue a task/script execution environment
+</pre>
+
+<pre class="anchors">
+spec: html; urlPrefix: https://html.spec.whatwg.org/
+    type: dfn
+        text: trusted; url: concept-events-trusted
+        text: browsing context; url: browsing-context
+
+spec: service-workers; urlPrefix: https://slightlyoff.github.io/ServiceWorker/spec/service_worker/index.html
+    type: dfn
+        text: active worker; url: dfn-active-worker
+        text: client; url: dfn-service-worker-client
+        text: control; url: dfn-control
+        text: extended lifetime promises; url: dfn-extend-lifetime-promises
+        text: frame type; url: dfn-service-worker-client-frame-type
+        text: handle functional event; url: handle-functional-event-algorithm
+        text: installing worker; url: dfn-installing-worker
+        text: service worker; url: service-worker-concept
+        text: service worker client; url: dfn-service-worker-client
+        text: service worker registration; url: service-worker-registration-concept
+        text: termination; url: terminate-service-worker-algorithm
+        text: waiting worker; url: dfn-waiting-worker
+    type: interface
+        text: ExtendableEvent; url: extendable-event-interface
+        text: ExtendableEventInit; url: extendable-event-init-dictionary
+        text: ServiceWorkerGlobalScope; url: service-worker-global-scope-interface
+        text: ServiceWorkerRegistration; url: service-worker-registration-interface
+</pre>
```

Comment:
A lot of these references aren't being used yet. Please introduce them together with their use.

Reply:
Better yet, never add them. Almost all cases where you're adding a <pre class="anchors"> you're doing it wrong, as ideally you should only be depending on terms that other specs are exporting (and that are indexed), eliminating the need for blocks like this.

Reply:
done. Note that I've had to add
<pre class=link-defaults>
to disambiguate or fix broken references.